### ISSUE TITLE
Support table function `numbers` and `numbers_mt`

### ISSUE THREAD

**Issue by sundy-li** [tensorbase/tensorbase#13]
It's not an easy way to try `tensorbase`.

The Blog [Hello, Base](https://tensorbase.io/2020/08/04/hello-base.html) has some docs about `nyc_taxi` datasets benchmarks with ClickHouse. Yet the dataset is pretty large, it's hard for users to explorer `tensorbase` quickly.

Maybe we can implement some table functions like `numbers` or `number_mt` in ClickHouse.

**Comment by jinmingjian** [comment]
good catch. though there is a tiny dataset (csv and bin) has been [shipped within the project](https://github.com/tensorbase/tensorbase/blob/master/samples)

**Comment by jinmingjian** [comment]
"numbers" is a kind of "fake" table, so be as a built-in style.  It is easy to add some ad-hoc logic to make this change in [codegen](https://github.com/tensorbase/tensorbase/blob/66a55f7a306c63f14d33b3eacf1e7ab83b96a83b/crates/ir/src/codegen.rs#L32).

Do you think so?:)

**Comment by jinmingjian** [comment]
I updated the [samples/README](https://github.com/tensorbase/tensorbase/tree/master/samples) for more infos to play with current Base.

**Comment by sundy-li** [comment]
How to check if it's imported correctly, maybe we should add AggregateFunction `count()`...

**Comment by jinmingjian** [comment]
Right. Now Base just supports sum agg. What you mentioned are just issue #2, issue #3 .
